realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -284,6 +288,16 @@ protected final void copy(ColumnInfo rawSrc, ColumnInfo rawDst) {
     }
 
     @Override
+    public RealmInteger realmGet$columnCounter() {
+        return new ManagedRealmInteger(0);
+    }
+
+    @Override
+    public void realmSet$columnCounter(RealmInteger value) {
+        throw new IllegalStateException("Cannot assign to a managed RealmInteger");
```

Comment:
We seem to be having trouble understanding progressive implementation here.  This is a feature branch.  Each commit moves it towards completion.  There is no intention that this is the complete implementation.